### PR TITLE
Css grid

### PR DESCRIPTION
**Pull request by tebanwolox**
## Summary
- Change tech structure with css-grid

## Screenshot
Screen Shot 2019-05-20 at 15 33 47

## Trello Card
https://trello.com/c/6Bd193if/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: scss/tools.scss [hunk 1]
```diff
@@ -6,8 +6,9 @@
   width: 100%;
 
   .techs {
-    display: flex;
+    display: grid;
     flex-wrap: wrap;
```
Comment (nmartinelli23) [inline scss/tools.scss]: No need of this.

File: scss/tech.scss [hunk 2]
```diff
@@ -34,20 +33,11 @@
...
-    z-index: -1;
-  }
+}
 
-  &:hover::before {
-    transform: translateX(10px) translateY(10px);
-  }
+.first-tech {
+  grid-row: 1/3;
+  justify-content: center;
```
Comment (nmartinelli23) [inline scss/tech.scss]: Same here.

File: scss/tech.scss [hunk 3]
```diff
@@ -34,20 +33,11 @@
...
-    transition: transform 0,6s linear;
-    width: 100%;
-    z-index: -1;
-  }
+}
 
-  &:hover::before {
-    transform: translateX(10px) translateY(10px);
-  }
+.first-tech {
```
Comment (henryzarza) [inline scss/tech.scss]: You don't need this, if you use `first-child` selector it works same

File: scss/tech.scss [hunk 4]
```diff
@@ -34,20 +33,11 @@
...
+  grid-row: 1/3;
+  justify-content: center;
+  justify-self: flex-end;
```
Comment (henryzarza) [inline scss/tech.scss]: I think that isn't necessary or i'm wrong?

File: scss/tools.scss [hunk 5]
```diff
@@ -6,8 +6,9 @@
...
     padding: 20px;
+    grid-template-columns: 2fr 1.5fr 1.5fr;
```
Comment (henryzarza) [inline scss/tools.scss]: Change this by: 
```suggestion
    grid-template-columns: 2fr 1fr 1fr;
```

File: scss/tech.scss [hunk 6]
```diff
@@ -34,20 +33,10 @@
...
-    z-index: -1;
-  }
-
-  &:hover::before {
-    transform: translateX(10px) translateY(10px);
+  &:first-child {
+    grid-row: 1/3;
+    justify-content: center;
+    justify-self: flex-end;
+    border-right: 1px solid $green;
```
Comment (henryzarza) [inline scss/tech.scss]: Remember order by alphabetically

**Pull request closed without merging**